docs/HowTo/Run-Docker-Image.md

```diff
@@ -0,0 +1,76 @@
+description: Run Orion using the official docker image
+<!--- END of page meta data -->
+
+# Running the Orion Docker Image
+
+Use the Orion Docker image to run a single Orion node without installing Orion.
+
+__Prerequisites__:
+
+* [Docker](https://docs.docker.com/install/)
+
+In this example, the [password file](#1-create-password-file), [public/private keypair](#2-generate-keys), and
+[Orion configuration file](#3-create-a-configuration-file) are created in the
+`/Users/user1/Node1/Orion` directory.
```

Comment:
note you have to create this dir first

Reply:
mkdir -p /Users/user1/Node1/Orion

Reply:
May be the document need to mention create "Node1/Orion" in user's home folder. On Mac, by default it is /User/_username_. On Linux it is usually /home/_username/

Reply:
I added a note about Mac vs Linux home directory and added a step to create the dir structure.